### PR TITLE
Adding badges and logo to README

### PR DESCRIPTION
**Pull request by guillaumebriday**
### Summary

Attempt to add colors and Rails logo in the README.

Live version here: https://github.com/guillaumebriday/rails/tree/patch-1

## Before
Screen Shot 2021-01-18 at 11 01 44

## After
Screen Shot 2021-01-18 at 11 01 24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+<p align="center">
+  <a href="https://rubyonrails.org/" target="_blank" rel="noopener noreferrer">
+    <img src="https://rubyonrails.org/images/rails-logo.svg" width="400">
+  </a>
+</p>
+
+<p align="center">
```
Comment (rafaelfranca) [inline README.md:7]: Can you remove all those badges? They are already in the contributing.md file given them are more relevant to contributors than users and the README file is targeted to users of the framework.

**Review by rafaelfranca — COMMENTED** [review]
I like the logo in the README but the badges have to go.

File: README.md (hunk 1 above)
Comment (rafaelfranca) [reply README.md:7]: Ah, they are not. Can you move them to there?

Comment (guillaumebriday) [reply README.md:7]: I can but I think it's cool for new comers to have a quick overview of these Rails "stats"

Comment (rafaelfranca) [reply README.md:7]: I understand, but those stats don't add much value or are already are in the page:

* master should be always green and make no different for new comers
* The number of downloads should not matter, those download are inflated by CI systems anyway
* The number of star is on the top of the repository page
* Let last version of the framework is on the sidebar on the Releases topic
* The License is also in the sidebar

Comment (guillaumebriday) [reply README.md:7]: You're right, will change

**Comment by guillaumebriday** [comment]
@rafaelfranca I updated the PR! 👍

**Comment by guillaumebriday** [comment]
@rafaelfranca and fixed the conflicts 👍

**Pull request merged**